src/modules/endpoint/endpoints/__tests__/disclaimer.spec.js

```diff
@@ -36,7 +36,7 @@ describe('disclaimer', () => {
 
   test('should map fetched data to models', () => {
     const disclaimerModel = disclaimer.mapResponse([pageJson])
-    expect(disclaimerModel).toEqual(new PageModel({
+    expect(disclaimerModel).toEqual(new CategoryModel({
```

Comment:
Is there already a task for DisclaimerModel?

Reply:
now it is :D